interview-questions.md

```diff
@@ -4,47 +4,47 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
+  Your answer: It is a method that takes in any number of state objects and allows them to be reffered to by shorthand for the duration of the current method you're in.
 
-  Researched answer:
+  Researched answer: After doing some research I discovered that destructing is a JavaScript feature that allows you to extract multiple pieces of data from an array or object and assign them to their own variables
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: A DOM event occurs when code allows a user input to update the DOM
```

Comment:
You pretty much got the right idea!  I know Sarah was able to clear this up a bit more for us in our review today.